man/fi_endpoint.3.md

```diff
@@ -566,6 +566,14 @@ The following option levels and option names and parameters are defined.
   each fi_trigger_var available, set the datatype and count of the variable
   used for the trigger.
 
+- *FI_OPT_CUDA_API_PERMITTED - bool \**
+: This option only applies to the fi_setopt call. It is used to control
+  endpoint's behavior in making calls to CUDA API. By default, an endpoint
+  is permitted to call CUDA API. If user wish to prohibit an endpint from
+  making such calls, user can achieve that by set this option to false.
+  If an endpoint's support of CUDA memory relies on making calls to CUDA API,
+  it will return -FI_EOPNOTSUPP for the call to fi_setopt.
```

Comment:
While we're in this file, can you update the page to say that fi_getopt/fi_setopt will return -FI_ENOPROTOOPT if the provider does not support the option?